src/klein/_dihttp.py

```diff
@@ -36,8 +34,6 @@ def urlFromRequest(request: IRequest) -> DecodedURL:
     else:
         host = request.client.host
         port = request.client.port
-        if not isinstance(host, text_type):
-            host = host.decode("ascii")
```

Comment:
is this always unreachable?

Reply:
I think so. The code path isn't hit by tests, at least not since we dropped Py2